src/taco-cli/cli/build.ts

```diff
@@ -137,7 +137,7 @@ class Build extends commands.TacoCommandBase {
         });
     }
 
-    private static build(commandData: commands.ICommandData): Q.Promise<tacoUtility.ICommandTelemetryProperties> {
+     protected runCommand(commandData: commands.ICommandData): Q.Promise<ICommandTelemetryProperties> {
```

Comment:
How about converting the other static methods in this class to instance methods as well - buildRemotePlatform, cleanPlatform?